h2o-algos/src/test/java/hex/tree/PrintMojoTreeTest.java

```diff
@@ -185,11 +186,30 @@ private void assertMojoJSONEqualsFixture(Model model, String fixtureFile) throws
     assertFalse(treeJson.isEmpty());
     final String expectedTreeJson = IOUtils.toString(getClass().getResourceAsStream(fixtureFile));
     assertEquals(
-        removeH2OVersion(expectedTreeJson), 
-        removeH2OVersion(treeJson)
+            removeH2OVersion(expectedTreeJson),
+            removeH2OVersion(treeJson)
     );
   }
   
+  private void assertMojoPngGenerated(Model model, String outputFileName) throws IOException {
```

Comment:
missing a test when the tree number is specified and instead of directory this creates just a plain file

Reply:
testMojoCategoricalPng() tests model with _ntrees = 1